ping/_compositions/rust.toml

```diff
@@ -1,29 +1,29 @@
 [master]
-CargoFeatures = 'libp2pmaster'
+BinaryName = 'testplan_0500'
 
 [custom]
-CargoFeatures = 'libp2pmaster'
+BinaryName = 'testplan_0500'
 
 [[groups]]
 Id = '0.49.0'
-CargoFeatures = 'libp2pv0490'
+BinaryName = 'testplan_0490'
 
 [[groups]]
 Id = '0.48.0'
-CargoFeatures = 'libp2pv0480'
+BinaryName = 'testplan_0480'
 
 [[groups]]
 Id = "v0.47.0"
-CargoFeatures = 'libp2pv0470'
+BinaryName = 'testplan_0470'
 
 [[groups]]
 Id = "v0.46.0"
```

Comment:
should here also be the `46.1`?

Reply:
Not sure, this is take verbatim from current master.